submissions/aymkin/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,172 @@
+            <div class="white-angle"></div>
+            <div class="grey-angle"></div>
+            <div class="popup">
+                <ul class="popup__list">
+                    <li class="popup__list-element">
+                        <a href="#" class="popup-a">
+                            <div class="sprite docs"></div>
+                            <div class="sprite-text">Docs</div>
+                        </a>
+                    </li>
+                    <li class="popup__list-element">
+                        <a href="#" class="popup-a">
+                            <div class="sprite coogle"></div>
+                            <div class="sprite-text">Google</div>
+                        </a>
+                    </li>
+                    <li class="popup__list-element">
+                        <a href="#" class="popup-a">
+                            <div class="sprite maps"></div>
+                            <div class="sprite-text">Maps</div>
+                        </a>
+                    </li>
+                    <li class="popup__list-element">
+                        <a href="#" class="popup-a">
+                            <div class="sprite youtube"></div>
+                            <div class="sprite-text">Youtube</div>
+                        </a>
+                    </li>
+                    <li class="popup__list-element">
+                        <a href="#" class="popup-a">
+                            <div class="sprite market"></div>
+                            <div class="sprite-text">Market</div>
+                        </a>
+                    </li>
+                    <li class="popup__list-element">
+                        <a href="#" class="popup-a">
+                            <div class="sprite news"></div>
+                            <div class="sprite-text">News</div>
+                        </a>
+                    </li>
+                    <li class="popup__list-element">
+                        <a href="#" class="popup-a">
+                            <div class="sprite drive"></div>
+                            <div class="sprite-text">Drive</div>
+                        </a>
+                    </li>
+                    <li class="popup__list-element">
+                        <a href="#" class="popup-a">
+                            <div class="sprite account"></div>
+                            <div class="sprite-text">Account</div>
+                        </a>
+                    </li>
+                    <li class="popup__list-element">
+                        <a href="#" class="popup-a">
+                            <div class="sprite blogger"></div>
+                            <div class="sprite-text">Blogger</div>
+                        </a>
+                    </li>
+                    <li class="popup__list-element">
+                        <a href="#" class="popup-a sprite forms">
+                            <div class="sprite forms"></div>
```

Comment:
Maybe, if you can use the class only for one of the tags (`a`, not `div`) corrected focus in display.